fee-hook/script/base/Config.sol

```diff
@@ -1,51 +1,118 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.19;
 
+import "forge-std/Script.sol";
+import "forge-std/StdJson.sol";
+
 /// @notice Shared configuration between scripts
-contract Config {
+contract Config is Script {
+    using stdJson for string;
+
     uint256 MAINNET = 1;
     uint256 ARBITRUM = 42161;
     uint256 BASE = 8453;
     uint256 ARB_SEP = 421614;
     uint256 SEP = 11155111;
+    uint256 POLYGON = 137;
 
     mapping(uint256 => address) public LZ_ENDPOINTS;
     mapping(uint256 => address) public POOL_MANAGERS;
     mapping(uint256 => address) public POSITION_MANAGERS;
+    mapping(uint256 => address) public V4_ROUTERS;
     mapping(uint256 => address) public VMOONEY_ADDRESSES;
     mapping(uint256 => address) public FEE_HOOK_ADDRESSES;
     mapping(uint256 => address) public TEST_TOKEN_ADDRESSES;
     mapping(uint256 => uint32) public LZ_EIDS;
+    mapping(uint256 => address) public CHAINLINK_ROUTERS;
+    mapping(uint256 => bytes32) public CHAINLINK_DONS;
+    mapping(uint256 => uint64) public CHAINLINK_SUBS;
+    mapping(uint256 => uint24) public LP_FEE;
 
     constructor() {
+        string memory ethJson = vm.readFile("../contracts/deployments/ethereum.json");
+        string memory arbJson = vm.readFile("../contracts/deployments/arbitrum.json");
+        string memory baseJson = vm.readFile("../contracts/deployments/base.json");
+        string memory sepJson = vm.readFile("../contracts/deployments/sepolia.json");
+        string memory polygonJson = vm.readFile("../contracts/deployments/polygon.json");
+        string memory arbSepJson = vm.readFile("../contracts/deployments/arbitrum-sepolia.json");
+
+        // vMOONEY doesn't exist on arbitrum-sepolia
```

Comment:
lol this file is getting wild. love it. can't wait to make a beautiful diagram of the topology of all these contracts and how they interact

Reply:
😍